frontend_tests/puppeteer_tests/08-admin.js

```diff
@@ -0,0 +1,380 @@
+"use strict";
+
+const assert = require("assert").strict;
+
+const common = require("../puppeteer_lib/common");
+
+async function submit_notifications_stream_settings(page) {
+    await page.waitForSelector('#org-submit-notifications[data-status="unsaved"]', {visible: true});
+
+    const save_button = "#org-submit-notifications";
+    assert.strictEqual(
+        await common.get_text_from_selector(page, save_button),
+        "Save changes",
+        "Save button has incorrect text.",
+    );
+    await page.click(save_button);
+
+    await page.waitForSelector('#org-submit-notifications[data-status="saved"]', {visible: true});
+    assert.strictEqual(
+        await common.get_text_from_selector(page, "#org-submit-notifications"),
+        "Saved",
+        "Saved text didn't appear after saving new stream notifications setting",
+    );
+
+    await page.waitForSelector("#org-submit-notifications", {hidden: true});
+}
+
+async function test_change_new_stream_notifications_setting(page) {
+    await page.click("#realm_notifications_stream_id_widget button.dropdown-toggle");
+    await page.waitForSelector("#realm_notifications_stream_id_widget ul.dropdown-menu", {
+        visible: true,
+    });
+
+    await page.type(
+        "#realm_notifications_stream_id_widget  .dropdown-search > input[type=text]",
+        "verona",
+    );
+    await page.click(
+        "#realm_notifications_stream_id_widget .dropdown-list-body > li:nth-of-type(1)",
+    );
+    await submit_notifications_stream_settings(page);
+
+    const disable_stream_notifications =
+        "#realm_notifications_stream_id_widget  .dropdown_list_reset_button";
+    await page.click(disable_stream_notifications);
+    await submit_notifications_stream_settings(page);
+}
+
+async function test_change_signup_notifications_stream(page) {
+    console.log('Changing signup notifications stream to Verona by filtering with "verona"');
+
+    await page.click("#id_realm_signup_notifications_stream_id > button.dropdown-toggle");
+    await page.waitForSelector(
+        "#realm_signup_notifications_stream_id_widget  .dropdown-search > input[type=text]",
+        {visible: true},
+    );
+
+    await page.type(
+        "#realm_signup_notifications_stream_id_widget  .dropdown-search > input[type=text]",
+        "verona",
+    );
+    await page.waitForSelector(
+        "#realm_signup_notifications_stream_id_widget  .dropdown-list-body > li.list_item",
+        {visible: true},
+    );
+    await page.keyboard.press("ArrowDown");
+    await page.keyboard.press("Enter");
```

Comment:
Used `keyboard` as `.click` was making the test flaky for no apparent reason and we'd also want t be testing the keyboard navigation too.( `.click` has been used for `stream` notifications above this)